scripts/create_test_customer.py

```diff
@@ -99,6 +100,8 @@ def create_customer(self):
         print("\nAdding customer...")
         response = requests.get(self.endpoint_base + "reports/aws/costs/", headers=get_headers(self.auth_token))
         print(f"Response: [{response.status_code}] {response.text}")
+        if response.status_code not in [200, 201]:
+            time.sleep(60)
```

Comment:
One thing I've noticed in my local environment is that creating the very first new customer takes so long that the API endpoint times out and returns a 504.

On the backend, the migrations are still running and will finish successfully, if given more time.

So, I've added some sleeps to work around this issue until a better solution is found.